mmv1/templates/terraform/examples/sql_postgres_instance_authorized_network.tf.erb

```diff
@@ -0,0 +1,18 @@
+# [START cloud_sql_postgres_instance_authorized_network]
+resource "google_sql_database_instance" "<%= ctx[:primary_resource_id] %>" {
+  name             = "<%= ctx[:vars]['postgres_instance_with_authorized_network'] %>"
+  region           = "us-central1"
+  database_version = "POSTGRES_12"
+  settings {
+    tier = "db-custom-2-7680"
+    ip_configuration {
+      authorized_networks {
+        name = "Network Name"
+        value = "192.0.2.0/24"
+        expiration_time = "3021-11-15T16:19:00.094Z"
+      }
+    }
+  }
+  deletion_protection =  "false"
```

Comment:
I think we want to use a variable to define this to differentiate the docs from the tests, similar to [here](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/examples/sql_database_instance_postgres.tf.erb#L9).

Reply:
done